espanol/bailar/salsa/salsa.md

```diff
@@ -0,0 +1 @@
+Alegre por librarse del imprudente consejero, saco a bailar a su esposa una salsita de Frankie Ruiz, el papá de la salsa Ok.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.